contents/uix/uix.md

```diff
@@ -2,6 +2,8 @@
 
 Author: [Ang Shi Ya](https://github.com/AngShiYa)
 
+Author: [Tan Jun Kiat](https://github.com/junkiattan)
+
```

Comment:
In other chapters it is given as `Authors: x, y`

Reply:
Noted and changed